Formula/l/lima.rb

```diff
@@ -39,7 +48,7 @@ def install
     info = JSON.parse shell_output("#{bin}/limactl info")
     # Verify that the VM drivers are compiled in
     assert_includes info["vmTypes"], "qemu"
```

Comment:
(`vmTypes` always includes "qemu`, regardless to whether QEMU is installed on the host)